### PR TITLE
allow ws(s):// in pac file resolves #3653

### LINKED ISSUE #3653: Lantern doesn't not proxy "ws://" and "wss://" scheme
should allow it in pac file

### PR DESCRIPTION
**Pull request by fffw**
Or the UI with special domain will not work correctly.

@xiam do you want to have a quick look?

### CODE REVIEW

File: src/github.com/getlantern/flashlight/main/pac.go [hunk 1]
```diff
@@ -90,8 +90,8 @@ func genPACFile() {
 					return "DIRECT";
 				}
 			}
-			// Lantern desktop version proxies only http and https
-			if (url.substring(0, 4) != 'http') {
+			// Lantern desktop version proxies only http(s) and ws(s)
+			if (url.substring(0, 4) != 'http' && (url.substring(0, 2) != 'ws')) {
```
Comment (fffw) [inline src/github.com/getlantern/flashlight/main/pac.go]: [List of registered schemes](http://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml). Whitelist seems more feasible.

**Comment by essichavoosh** [comment]
II need theget lanter break filtering for my self

**Comment by oxtoacart** [comment]
@fffw Good catch.

**Comment by fffw** [comment]
@oxtoacart Do you think it's good to be merged?

**Comment by oxtoacart** [comment]
LGTM!

**Pull request merged**